src/scheduler/job_info.c

```diff
@@ -1037,7 +1037,7 @@ query_jobs(status *policy, int pbs_sd, queue_info *qinfo, resource_resv **pjobs,
 	resresv_arr[num_prev_jobs] = NULL;
 
 	tid = *((int *) pthread_getspecific(th_id_key));
-	if (tid != 0 || num_threads == 1) {
+	if (tid != 0 || num_threads <= 1) {
```

Comment:
Now that you've floored num_threads to 1, it's up to you if you still want to make all the cmp changes.

Reply:
I think it's safer to have them check for <= so I'll leave them be.